### PR TITLE
Qsw 862 - Dashboard

### PR DESCRIPTION
**Pull request by claudio525**

### CODE REVIEW

File: dashboard/DashboardDB.py [hunk 1]
```diff
@@ -116,7 +155,48 @@ def update_squeue(self, squeue_entries: Iterable[SQueueEntry], hpc: const.HPC):
...
+                    (
+                        entry.id,
+                        entry.name,
+                        entry.int_value_1,
+                        entry.int_value_2,
+                        datetime.now(),
+                    ),
+                )
 
     def get_sqeue_entries(self, hpc: const.HPC):
```
Comment (jasonmotha) [inline dashboard/DashboardDB.py]: spelling

File: dashboard/run_data_collection.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+    # Update dashboard db
+    if rt_ch_output:
+        dashboard_db.update_daily_chours_usage(
+            parse_total_chours_usage(rt_ch_output, hpc), hpc
+        )
+    else:
+        return False
+
+    # Squeue
+    sq_output = run_cmd(user, hpc.value, "squeue")
```
Comment (jasonmotha) [inline dashboard/run_data_collection.py]: -A nesi00213

Comment (claudio525) [reply dashboard/run_data_collection.py]: I recon log everything for now (have to include the account in the results though) and then add a filter option to the table, which I think dash makes quite easy

File: dashboard/DashboardDB.py (hunk 1 above)
Comment (claudio525) [reply dashboard/DashboardDB.py]: fixed

**Pull request merged**